### PR TITLE
Do not show the expression info when contains secret

### PR DESCRIPTION
**Pull request by liangxia**
/cc @jhou1 @JianLi-RH @dis016 @pruan-rht

### CODE REVIEW

File: features/step_definitions/common.rb [hunk 1]
```diff
@@ -141,19 +141,23 @@
       comparison_operators = /===|==|>=|<=|!=|>|<|&&|\|\|&|\|/
       op = expr.scan(comparison_operators).first
       if op
-        e1, e2 = expr.split(op)
+        expr_left, expr_right = expr.split(op)
         # XXX: we may not catch everything, wrap it around begin/rescue and print
         # out blank if some complex comparison is in the expression.
         begin
-          eval_details = "\nleft_operand: #{eval(e1)}, right_operand: #{eval(e2)}\n"
+          if ( (expr_left.include? "secret") || (expr_right.include? "secret") )
```
Comment (jhou1) [inline features/step_definitions/common.rb:148]: Maybe adding a list of keywords to the watch and masking them. Such as `secrets, token, key, creds, credential, password etc.`
Suggest to use regexp to also capture upper and lower cases

Comment (liangxia) [reply features/step_definitions/common.rb]: Fixed.

**Pull request merged**